### DISCUSSION TITLE
support rspack?

### DISCUSSION THREAD

**Discussion by legend-li** [posva/unplugin-vue-router#352] [Questions and Help] [answered]
can unplugin-vue-router support rspack?

**Comment by posva** [comment, marked as answer]
Anything supported by https://github.com/unjs/unplugin will be supported. Rspack seems to have experimental support. They can all be imported with `import { thing } from 'unplugin-vue-router'` (replace thing with rspack, vite, etc)

**Reply by legend-li** [reply]
I find use the resolveId hook in the unplugin-vue-router source code, Is it mean that unplugin-vue-router does not support Rspack environment. because unplugin does not  support resolveId hook in Rspack environment.

![image](https://github.com/posva/unplugin-vue-router/assets/20475084/8ed69f75-1ec4-4660-9b27-7ddf7c9c249e)
![image](https://github.com/posva/unplugin-vue-router/assets/20475084/e1c10769-195f-431d-966b-d2e46f5290c1)
